sdk/nodejs/provider.ts

```diff
@@ -1508,14 +1508,18 @@ export namespace admissionregistration {
 
 
       /**
-       * Get an existing InitializerConfiguration resource's state with the given name, ID, and optional extra
-       * properties used to qualify the lookup.
+       * Get the state of an existing `InitializerConfiguration` resource, as identified by `id`.
+       * Typically this ID  is of the form <namespace>/<name>; if <namespace> is omitted, then (per
+       * Kubernetes convention) the ID becomes default/<name>.
        *
-       * @param name The _unique_ name of the resulting resource.
-       * @param id The _unique_ provider ID of the resource to lookup.
+       * Pulumi will keep track of this resource using `name` as the Pulumi ID.
+       *
+       * @param name _Unique_ name used to register this resource with Pulumi.
+       * @param kubeId An ID for the Kubernetes resource to retrive. Takes the form
+       *  <namespace>/<name> or <name>.
        */
-      public static get(name: string, id: pulumi.Input<pulumi.ID>): InitializerConfiguration {
-          return new InitializerConfiguration(name, undefined, { id });
+      public static get(name: string, kubeId: pulumi.Input<pulumi.ID>): InitializerConfiguration {
+          return new InitializerConfiguration(name, undefined, { id: kubeId });
```

Comment:
I think this was PR feedback based on a comment I made - but I don't actually think we should change `id` to `kubeId`.  My comment was about whether the string `"The _unique_ provider ID"` was the correct terminology to use in the comment - I think the new comment is an improvement, but doesn't require changing the parameter name.